test/routes/src/server/routes/t_store.js

```diff
@@ -0,0 +1,38 @@
+import expect from 'expect';
+import Express from 'express';
+import nock from 'nock';
+import supertest from 'supertest';
+
+import store from '../../../../../src/server/routes/store';
+import { conf } from '../../../../../src/server/helpers/config';
+
+describe('The store API endpoint', () => {
+  const app = Express();
+  app.use(store);
+
+  describe('register-name route', () => {
+    afterEach(() => {
+      nock.cleanAll();
+    });
+
+    it('passes request through to store', (done) => {
```

Comment:
Could be worth having test for error response.

Especially if we end up wrapping it in `{ success, payload }` scheme. Right now I agree that it may be not necessary as it's just a pass-through.